src/OrchardCore.Modules/OrchardCore.Settings/SiteSettings.cs

```diff
@@ -1,10 +1,14 @@
+using System;
+using System.Collections.Generic;
 using Microsoft.AspNetCore.Routing;
+using Newtonsoft.Json.Linq;
 using OrchardCore.Documents;
+using OrchardCore.Entities;
 
 namespace OrchardCore.Settings
 {
     // When updating class also update SiteSettingsDeploymentSource and SettingsStep.
-    public class SiteSettings : DocumentEntity, ISite
+    public class SiteSettings : DocumentEntity, ICacheableEntity, ISite
```

Comment:
I think you need to let `DocumentEntity` implements `ICacheableEntity` instead, so all the implementation will be no long exist here

Reply:
Not sure we want that behavior applied everywhere. If that is the cache, we can just remove `ICacheableEntity` and move the new methods into `IEntity` instead.